README.md

```diff
@@ -62,78 +62,92 @@ genericLogger.indexDocument(document, function(error, body) {
 
 ### With Messenger
 
-```javascript
-// Messenger bot response format
-{
-  recipient: {
-    id: "recipientId"
-  },
-  message: {
-    text: "hello, world!"
-  }
-}
-```
+Log the message inside the POST request callback made when the bot replies (the user message and bot response(s) must be logged in the same document) (see right pane).
+
+<a type="button" target="_blank" class="button" href="https://github.com/Botfuel/sample-bot-messenger">
```

Comment:
This looks a bit strange on github I think 🤣

Reply:
You think? 😆 
![image](https://user-images.githubusercontent.com/3613657/30164339-78d02542-93dc-11e7-8667-1dcb3cbe7663.png)

Reply:
Done